spec/requests/create_technical_metadata_spec.rb

```diff
@@ -13,9 +13,11 @@
   end
   let(:payload) { { sub: 'sdr' } }
   let(:jwt) { JWT.encode(payload, Settings.hmac_secret, 'HS256') }
+  let(:job) { double }
```

Comment:
can this be a `class_double(TechnicalMetadataWorkflowJob)`?

Reply:
Yup. Fixed.